.jenkinsci/docker-cleanup.groovy

```diff
@@ -1,17 +1,23 @@
 #!/usr/bin/env groovy
 
+// remove docker network and stale images
 def doDockerCleanup() {
+  sh """
+    # Check whether the image is the last-standing man
+    # i.e., no other tags exist for this image
+    docker rmi \$(docker images --no-trunc --format '{{.Repository}}:{{.Tag}}\\t{{.ID}}' | grep \$(docker images --no-trunc --format '{{.ID}}' ${iC.id}) | head -n -1 | cut -f 1) || true
+  """
+}
+
+// cleanup docker network created for the test stage
+def doDockerNetworkCleanup() {
+  sh "docker network rm ${env.IROHA_NETWORK}"
+}
 
-    sh """
-      # Check whether the image is the last-standing man
-      # i.e., no other tags exist for this image
-      docker rmi \$(docker images --no-trunc --format '{{.Repository}}:{{.Tag}}\\t{{.ID}}' | grep \$(docker images --no-trunc --format '{{.ID}}' ${iC.id}) | head -n -1 | cut -f 1) || true
-      sleep 5
-      docker network rm $IROHA_NETWORK || true
-      #remove folder with iroha.deb package and Dockerfiles
-      rm -rf /tmp/${env.GIT_COMMIT}-${BUILD_NUMBER}
-      rm -rf /tmp/${env.GIT_COMMIT}
-    """
+// cleanup docker images which weren't used for more that 20 days and image for this PR in case of successful PR
+def doStaleDockerImagesCleanup() {
+  sh "find ${JENKINS_DOCKER_IMAGE_DIR} -type f -mtime +20 -exec rm -f {}"
```

Comment:
This fails with an error on Ubuntu 16.04. Did you check this really works? I think it should end with ` \;` after curly braces.

Reply:
@bakhtin Thanks for the report, I've missed `\;`. I've just checked it on the real files (AWS EFS) and it WORKS!